### PR TITLE
Myriadcoin and Unobtanium

### PR DESCRIPTION
**Pull request by cryptapus**

### CODE REVIEW

File: src/networks.js [hunk 1]
```diff
@@ -124,6 +124,32 @@ var networks = {
...
+    estimateFee: estimateFee('myriadcoin')
+  },
+  unobtanium: {
+    magicPrefix: '\x18Unobtanium Signed Message:\n',
+    bip32: {
+      public: 0x0488b21e, // https://github.com/unobtanium-official/Unobtanium/pull/8/files
+      private: 0x0488ade4 // https://github.com/unobtanium-official/Unobtanium/pull/8/files
+    },
+    pubKeyHash: 0x82,
+    scriptHash: 0x1e,
```
Comment (dcousens) [inline src/networks.js]: same as dogecoin `pubkeyhash`

File: src/networks.js [hunk 2]
```diff
@@ -124,6 +124,32 @@ var networks = {
     dustThreshold: 546, // https://github.com/zetacoin/zetacoin/blob/master/src/core.h#L159
     feePerKb: 10000, // https://github.com/zetacoin/zetacoin/blob/master/src/main.cpp#L54
     estimateFee: estimateFee('zetacoin')
+  },
+  myriadcoin: {
+    magicPrefix: '\x18Myriadcoin Signed Message:\n',
+    bip32: {
```
Comment (dcousens) [inline src/networks.js]: warning: same constants as bitcoin

File: src/networks.js [hunk 3]
```diff
@@ -124,6 +124,32 @@ var networks = {
...
+    pubKeyHash: 0x32,
+    scriptHash: 0x09,
+    wif: 0xb2,
+    dustThreshold: 546, // https://github.com/myriadcoin/myriadcoin/blob/master/src/core.h#L220
+    feePerKb: 10000, // https://github.com/myriadcoin/myriadcoin/blob/master/src/main.cpp#L53
+    estimateFee: estimateFee('myriadcoin')
+  },
+  unobtanium: {
+    magicPrefix: '\x18Unobtanium Signed Message:\n',
+    bip32: {
```
Comment (dcousens) [inline src/networks.js]: warning: same constants as bitcoin

File: src/networks.js [hunk 4]
```diff
@@ -124,6 +124,32 @@ var networks = {
...
     estimateFee: estimateFee('zetacoin')
+  },
+  myriadcoin: {
+    magicPrefix: '\x18Myriadcoin Signed Message:\n',
+    bip32: {
+      public: 0x0488b21e,
+      private: 0x0488ade4
+    },
+    pubKeyHash: 0x32,
+    scriptHash: 0x09,
```
Comment (dcousens) [inline src/networks.js]: same as zetacoin `scriptHash`

File: src/networks.js [hunk 5]
```diff
@@ -124,6 +124,32 @@ var networks = {
...
+  },
+  unobtanium: {
+    magicPrefix: '\x18Unobtanium Signed Message:\n',
+    bip32: {
+      public: 0x0488b21e, // https://github.com/unobtanium-official/Unobtanium/pull/8/files
+      private: 0x0488ade4 // https://github.com/unobtanium-official/Unobtanium/pull/8/files
+    },
+    pubKeyHash: 0x82,
+    scriptHash: 0x1e,
+    wif: 0xe0,
```
Comment (dcousens) [inline src/networks.js]: same as zetacoin `wif`

**Comment by dcousens** [comment]
These networks will almost never be automatically detected, see above comments, https://github.com/bitcoinjs/bitcoinjs-lib/issues/325 and https://github.com/bitcoinjs/bitcoinjs-lib/issues/278.

**Comment by dcousens** [comment]
@weilu thoughts?

**Comment by dcousens** [comment]
@weilu @caedesvvv thoughts?

**Comment by weilu** [comment]
If the OP doesn't address the above comments in 3 days we close this without merging.

**Comment by cryptapus** [comment]
Noted that these coins have duplicate values to what is already in networks.js and is an issue. Until I have time to investigate a more elegant solution I propose that it be closed.

Regards,

**Pull request closed without merging**